### ISSUE TITLE
Error creating STOP_MARKET order for future api? 

### ISSUE THREAD

**Issue by Swoorup** [paoloboni/binance-scala-client#314]
```scala
client
  .use(h =>
    h.createOrder(
      FutureOrderCreateParams.STOP_MARKET(
        symbol        = "BTCUSDT",
        side          = OrderSide.BUY,
        positionSide  = FuturePositionSide.BOTH,
        stopPrice     = 30000.0,
        closePosition = false,
        priceProtect  = false,
        reduceOnly    = false,
      )
    )
  )
```

Currently getting the following error. Could it be due to missing quantity field? 

```shell
sttp.client3.HttpError: statusCode: 400, response: {"code":-1102,"msg":"A mandatory parameter was not sent, was empty/null, or malformed."}
	at sttp.client3.HttpError$.apply(ResponseAs.scala:228)
	at sttp.client3.ResponseAs$.deserializeRightWithError$$anonfun$1(ResponseAs.scala:180)
	at sttp.client3.MappedResponseAs.mapWithMetadata$$anonfun$1(ResponseAs.scala:92)
	at sttp.client3.internal.BodyFromResponseAs.doApply$$anonfun$2(BodyFromResponseAs.scala:24)
	at map @ io.github.paoloboni.binance.fapi.FutureApi.createOrder$$anonfun$1$$anonfun$1(FutureApi.scala:256)
	at rethrow$extension @ fs2.Compiler$Target.compile$$anonfun$1(Compiler.scala:157)
	at modify @ fs2.internal.Scope.close(Scope.scala:262)
	at flatten @ sttp.client3.impl.cats.CatsMonadError.flatten(CatsMonadError.scala:21)
	at offer @ io.github.paoloboni.http.ratelimit.RateLimiter$$anon$1.rateLimit$$anonfun$1(RateLimiter.scala:54)
	at *> @ io.github.paoloboni.http.ratelimit.package$LimiterOps.await(package.scala:35)
```

**Comment by Swoorup** [comment]
Can confirm quantity is the reason.

**Comment by paoloboni** [comment]
HI @Swoorup, is this solved in [v1.5.3](https://github.com/paoloboni/binance-scala-client/releases/tag/v1.5.3)?
